config/v1/feature_gates.go

```diff
@@ -202,4 +202,14 @@ var (
 		ResponsiblePerson:   "miciah",
 		OwningProduct:       ocpSpecific,
 	}
+
+	FeatureGateImagePolicy = FeatureGateName("ImagePolicy")
```

Comment:
I think we make it more descriptive, perhaps `sigstoreImageVerification`

Reply:
Good, changed as suggested.